Documentation/design-docs/roll-forward-on-no-candidate-fx.md

```diff
@@ -0,0 +1,86 @@
+# Roll Forward On No Candidate Fx
+
+## Before changes
+
+The desired framework version is defined through a configuration json file (appname.runtimeconfig.json). For productions, the default behavior is to pick the latest patch compatible available version.
+
+	For instance:
+
+	Desired version: 1.0.1
+	Available versions: 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.1.0, 1.1.1, 2.0.1
+	Chosen version: 1.0.3
+
+	Desired version: 1.0.1
+	Available versions: 1.0.0, 1.1.0, 2.0.0
+	Chosen version: there is no compatible version available
+
+It's possible to disable the patch roll forward through the "applyPatches" property in the configuration file. If it is  set to 'false' and the specified version is not found, then we fail.
+
+	For instance:
+
+	Patch roll forward: disabled
+	Desired version: 1.0.1
+	Available versions: 1.0.2, 1.0.3
+	Chosen version: there is no compatible version available
+
+It's also possible to specify the desired framework version through the command line argument '--fx-version'. In this case, only the specified version will be accepted, even if patch roll forward is enabled. The expected behavior would be the same in the example above.
+
+## After changes
+
+### Roll Forward On No Candidate Fx
```

Comment:
Make this text more user friendly to read - "Roll Forward in Absence of Candidate FX"